android/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
+distributionBase="GRADLE_USER_HOME"
+distributionPath="wrapper/dists"
+distributionUrl="https\\://services.gradle.org/distributions/gradle-7.5.1-all.zip"
```

verifique se precisa do \\ barra contra na url
```suggestion
distributionUrl="https\\://services.gradle.org/distributions/gradle-7.5.1-all.zip"
```